packages/connect-voting-disputable/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@aragon/connect-voting-disputable",
```

Comment:
Should we use the name `@aragon/connect-disputable-voting`? It seems to be a more common way to refer to it.